db/migrate/20230819021905_create_users.rb

```diff
@@ -0,0 +1,14 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.text :photo
```

Comment:
- [ ] Hi @mohamedSabry0, data type for` photo` should be `string` not `text` as `photo `should be a link to an image

Reply:
Hi @FrederickMih,

Thank you for the review. As for the type, since we are not sure of the URL length it would be better to go with the safe choice. That was my thought process when I chose a text type instead of a string. What are your thoughts on this part?

Reply:
Hi @mohamedSabry0, I don't think the length of the URL matters. We are told to use a string. It's important to adhere to the requirements of our projects. You can always add your thoughts after making sure you have respected the project's requirements.